services/src/main/resources/lax-security-profile.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name":"test",
```

Comment:
Is name `test` ok?

Reply:
Nope, it was a test before, moved to `lax`.